src/chart/parameter/component/NodePropertyParameterSelect.tsx

```diff
@@ -11,10 +11,23 @@ const NodePropertyParameterSelectComponent = (props: ParameterSelectProps) => {
       ? props.settings.defaultValue
       : '';
 
+  const getInitialValue = (value, multi) => {
```

Comment:
Noticing a strange bug when doing the following:
- create a parameter selector for node properties. leave everything default
- save the card and choose a value
- open the card and turn on multiselect
- choose another value
Then this happens:
![image](https://github.com/neo4j-labs/neodash/assets/1754968/0aae8d80-a387-4f97-abfe-27c2a5c59d15)

(It converts every character in the selected value as an array element. probably it's confused about the types when switching between selectors.

Reply:
fixed